cypress/integration/admin/login/login.spec.js

```diff
@@ -17,4 +19,146 @@ context("Login Page", () => {
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.findByLabelText(/your e-mail/i).type("admin@webiny.com");
+        cy.findByLabelText(/your password/i).type("12345678");
+        cy.findByText(/submit/i).click();
+
+        cy.findByPlaceholderText(/search.../i).should("exist");
+        cy.findByText(/pages/i).should("exist");
+
+        cy.visit("/page-builder/pagessss");
+        cy.findByText(/The route is either missing, or you're not authorized to view it./i).should(
+            "exist"
+        );
+        cy.findByText(/Please contact your administrator to report the issue./i).should("exist");
+        cy.findByText(/Take me back./i).should("exist");
+    });
+
+    it("authenticated user must logout successfully", () => {
+        cy.visit("/");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.findByLabelText(/your e-mail/i).type("admin@webiny.com");
+        cy.findByLabelText(/your password/i).type("12345678");
+        cy.findByText(/submit/i).click();
+
+        cy.findByPlaceholderText(/search.../i).should("exist");
+        cy.findByText(/pages/i).should("exist");
+
+        cy.findByTestId("logged-in-user-menu-avatar").click();
+        cy.findByText(/Sign out/i).click();
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+    });
+
+    it('recover password E2E for "full-access" user', () => {
+        const serverId = 'z1fihlo8';
+        let passwordResetCode;
+        const newUserAccountEmail = uniqid("cypress_", "@z1fihlo8.mailosaur.net");
+
+        let fullAccessGroupUser;
+        let password = "12345678";
+        let newPassword = "12345678910"
+
+        // eslint-disable-next-line jest/valid-expect-in-promise
+        cy.securityReadGroup({ slug: "full-access" }).then(group => {
+            return cy
+                .securityCreateUser({
+                    data: {
+                        email: newUserAccountEmail,
+                        firstName: uniqid("first name-"),
+                        lastName: uniqid("last name-"),
+                        password,
+                        group: group.id
+                    }
+                })
+                .then(user => {
+                    fullAccessGroupUser = user;
+                    // Login with new user
+                    cy.login({ username: fullAccessGroupUser.email, password });
+                    cy.log('email:' + fullAccessGroupUser.email)
+                    cy.log('psw:' + password)
+                    cy.visit("/");
+                    // Verify the access
+                    cy.findByText(`Hi ${user.firstName} ${user.lastName}!`).should("be.visible");
+                    cy.findByText(/To get started - pick one of the actions below:/i).should(
+                        "be.visible"
+                    );
+
+                    // Should have Page Builder card
```

Comment:
Let's remove these couple of steps - we've already tested that in another test, and we've also done the "To get started..." assertion. I'd sad removing this won't hurt us.

Reply:
I let this assert here on purpose in order to make sure that the test waits until the page is loaded so it can securely click the user avatar in the top-right corner - sometimes cypress is a bit lazy and just try to click elements even though the page is not completely loaded. However I am going to delete it and we can add it later if we see something strange at this point